lib/utils.js

```diff
@@ -353,7 +353,7 @@ async function getPIDsListeningOnPort (port, filteringFunc = null) {
     const {stdout} = await exec('lsof', ['-ti', `tcp:${port}`]);
     result.push(...(stdout.trim().split(/\n+/)));
   } catch (e) {
-    log.debug(e);
+    log.debug(`Error getting processes listening on port '${port}': ${e.message}`);
```

Comment:
I would check the `e.code` value. In case it equals to 1 then no processes are found (which is fine and expected) otherwise print this message

Reply:
Good idea.